src/resolve/resolved-link.c

```diff
@@ -617,6 +638,10 @@ static void link_read_settings(Link *l) {
         r = link_update_search_domains(l);
         if (r < 0)
                 log_warning_errno(r, "Failed to read search domains for interface %s, ignoring: %m", l->name);
+
+        r = link_update_default_route(l);
+        if (r < 0)
+                log_warning_errno(r, "failed to read default route setting for interface %s, ignoring: %m", l->name);
```

Comment:
"Failed"

Also, "ignoring" is not entirely appropriate, since it's being reset anyway.

Reply:
will change to "proceeding anyway"